test/node.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-env mocha */
+'use strict'
+
+require('./connect')
```

Comment:
What do you think about organizing these files a bit more? I am thinking something like:
```
connect/
  -- index.js (requires all the connect tests)
  -- js2go.js
  -- go2go.js
  -- go2js.js
```

I think it might keep things a bit cleaner. The ipfs/interop tests are a bit messy to find things. This might help with that. It wouldn't help a whole lot with 3+ nodes which we'll hit pretty frequently with dht/pubsub/etc, but perhaps we can find ways of keeping that clean.


Either way, this looks good so far 👍.  Are you using or planning to use the PDD scenario files to create the tests? I think documenting the scenarios is a good thing, but perhaps the PDDs that have scenarios are a bit out of day and would warrant replacing altogether.

Reply:
My biggest concern right now with that structure is in the cases that we may want to use a common `before`. But I think that what we may need in the `before` should be simple, and we can also have utils for that, so we can go with your proposal.

Regarding the PDD, I am not using the tests but I decided to keep them for now, as they may be useful. Either way, we can possibly do:

```
connect/
  -- index.js (requires all the connect tests)
  -- js2go.js
  -- go2go.js
  -- go2js.js
  -- README.md (document the tests)
dht/
  -- content /
    -- js2go.js
    -- go2go.js
    -- go2js.js
  -- provide /
    -- js2go.js
    -- go2go.js
    -- go2js.js
```